deis-logger-beta1-test/manifests/deis-logger-svc.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: deis-logger
+  namespace: deis
+  labels:
+    heritage: deis
+    app: deis-logger
+spec:
+  ports:
+  - port: 80
+    name: http
+    targetPort: http
+  - port: 514
+    name: transport
+    targetPort: 1514
+    protocol: UDP
+  selector:
+    app: deis-logger
```

Comment:
Some people hate them others want them. I don't even notice them to be
honest.
On Mar 18, 2016 5:55 PM, "Aaron Schlesinger" <notifications@github.com>
wrote:

> In deis-logger-beta1-test/manifests/deis-logger-svc.yaml
> <https://github.com/deis/charts/pull/180#discussion_r56727887>:
>
> > +  name: deis-logger
> > +  namespace: deis
> > +  labels:
> > +    heritage: deis
> > +    app: deis-logger
> > +spec:
> > +  ports:
> > +  - port: 80
> > +    name: http
> > +    targetPort: http
> > +  - port: 514
> > +    name: transport
> > +    targetPort: 1514
> > +    protocol: UDP
> > +  selector:
> > +    app: deis-logger
>
> Are we putting newlines at the end of files? I prefer yes, but no strong
> feelings. This PR is heterogeneous though.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/deis/charts/pull/180/files/47cfecea316998615f3499cc4d042d223db814fa#r56727887>
>